geps/gep-1426.md

```diff
@@ -467,3 +467,10 @@ An initial iteration of this GEP had the ability to omit a `backendRef` and have
 This has been removed due to inconsistency with Gateway `parentRefs` and tight coupling of the "frontend" and "backend" roles.
 
 Implementations MUST respect the standard `backendRef` rules as defined by the existing spec.
+
+## Graduation Criteria
+
+The following are items that **MUST** be resolved to move this GEP to
+`Standard` status (and before the end of the probationary period):
+
+...
```

Comment:
Same as above, leaving a comment so we don't miss this.

Reply:
GEP-1686 (geps/gep-1686.md) is in the same boat, yup. We need to sort that out this week.

Reply:
Likewise fixed.